cmd/wallet/main.go

```diff
@@ -44,6 +49,11 @@ func main() {
 		cmd.PactusDefaultWalletPath(cmd.PactusDefaultHomeDir()), "the path to the wallet file")
 	offlineOpt = rootCmd.PersistentFlags().Bool("offline", false, "offline mode")
 	serverAddrOpt = rootCmd.PersistentFlags().String("server", "", "server gRPC address")
+	timeoutOpt = rootCmd.PersistentFlags().Int("timeout", 1,
+		"specifies the timeout duration for the client connection in seconds. "+
+			"this timeout determines how long the client will attempt to establish a connection with the server "+
```

Comment:
It is too long. How the `-h` looks like after this change?
Do we have "Long Help" option in cobra?

Reply:
I checked the Cobra. The 3rd argument in Options is `Usage`. So if we need to add Long Help we should add it into the command. It supports Long Help .

Reply:
> I checked the Cobra. The 3rd argument in Options is `Usage`. So if we need to add Long Help we should add it into the command. It supports Long Help .

This is switch not flag, just have usage.